template-compose/detekt-config.yml

```diff
@@ -286,7 +287,7 @@ style:
     active: true
     ignoreNumbers: '-1,0,1,2'
```

Comment:
@Wadeewee Shall we also add: `ignoreAnnotated: [ 'Preview' ]` for `MagicNumber`? 🙏 

For example, if we have a composable with an `Int` parameter:

```
@Composable
fun AppBar(height: Int) {
    TopAppBar(
        modifier = Modifier.height(height)
    )
}
```

In some cases, we would want to pass a raw integer value to the preview:

```
@Preview(showBackground = true)
@Composable
private fun AppBarPreview() {
    AppBar(height = 200)
}
```

What do you think? We did the same for [dee-money migration](https://github.com/nimblehq/dee-money-android/pull/34/files)

Reply:
@Tuubz It's good to handle this for `Preview` in `MagicNumber` but I think we should consider using `PreviewParameterProvider` instead So I think we can get rid of this 🤔 

Also, From your example Is it can be possible right -> `AppBar(height = 200)` 🤔  because it should be suffix with  `.dp` -> `AppBar(height = 200.dp)` What do you think?

Reply:
@Tuubz My Detekt does not report it with the current config. Am I missing something from your concern?

![image](https://user-images.githubusercontent.com/16315358/208586738-06675298-8c3f-46ae-82fe-3fd268f13bc0.png)

Reply:
@Tuubz Could you please give me more some example that can be happen for passing `int` to `Preview` 🙏

Reply:
@Tuubz I made another example and it still does not report the issue 🤔 cc @Wadeewee 

![image](https://user-images.githubusercontent.com/16315358/208587680-b9b71c08-d627-49c7-a8e0-438e6c3e503d.png)

Reply:
Well, this is embarrassing, I can't reproduce it either 😂 🙈 

Sorry for wasting your time, let's resolve this